src/main/java/gov/nih/nci/evs/api/fhir/R4/CodeSystemProviderR4.java

```diff
@@ -101,7 +103,13 @@ public Parameters lookupImplicit(
       @OperationParam(name = "displayLanguage") final StringType displayLanguage,
       @OperationParam(name = "property") final CodeType property)
       throws Exception {
-
+    // check if request is a post, throw exception as we don't support post calls
```

Comment:
Optional, but it is probably better to add a custom annotation to deal with this rather than repeating this code in other places. Like a @POST

Reply:
Oh, that's a good idea. We can implement that as an optimization or improvement. I'll create a ticket for that so we do that